next.clayui.com/gatsby/createPages.js

```diff
@@ -6,13 +6,17 @@
 
 'use strict';
 
+/* eslint-disable liferay/imports-first */
+
 require('dotenv').config({
 	path: `.env.${process.env.NODE_ENV}`,
 });
 
 const componentWithMDXScope = require('gatsby-mdx/component-with-mdx-scope');
 const path = require('path');
 
+/* eslint-enable liferay/imports-first */
```

Comment:
This is because `dotenv` is a special beast; it is recommended to call `config()` on it as early as possible in the app boot-up sequence, but we have lints against putting imports after non-import statements (and the `config()` call is considered a non-import statement), so I added this suppression.

I don't know if we even need dotenv though because I don't think anybody is creating `.env` files, right?

Reply:
This will load the env: `.env.development` and `.env.production`.

Reply:
Yeah, I know what "dotenv" does — I was just asking, does anyone on our team (or any tool that we're using) — actually use those files?

Reply:
And looking on the filesystem, I know see that we do have those files in a couple places; when I first looked for them I must have been looking in the repo root instead of under "next.clayui.com/".

Reply:
We have these two files at `next.clayui.com/`, to define a "Nightly"environment that prevents creating pages for production and only being enabled in the dev environment, it was a feature that was doing to migrate all Patrick docs to the clayui.com website but without some being visible in the production. But unsuccessfully getting Patrick out of there 😅